.github/ISSUE_TEMPLATE/project-triaging.yaml

```diff
@@ -0,0 +1,90 @@
+name: Project Triaging
+description: A template for information to look for in an NGI project's resources
+title: <PROJECT_NAME>
+labels: ["project"]
```

Comment:
```suggestion
labels: ["NGI Project"]
```

Reply:
The label is currently called [project](https://github.com/ngi-nix/ngipkgs/labels/project). Are there any plans to rename it? Will there be other types of projects?

Reply:
It is more intuitive than just `project`.

Reply:
To me this seems rather redundant if we're not distinguishing it from other projects, but we should avoid breaking things in all cases.